apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/klienter/SamKlient.kt

```diff
@@ -89,20 +91,24 @@ internal fun Vedtak.tilSamordneRequest(): SamordneVedtakRequest {
     return SamordneVedtakRequest(
         pid = this.soeker,
         vedtakId = this.id,
+        sakId = this.sakId,
         virkFom = innhold.virkningstidspunkt.atDay(1),
         virkTom = innhold.utbetalingsperioder.maxByOrNull { it.periode.fom }?.periode?.tom?.atEndOfMonth(),
         fagomrade = "EYO",
         ytelseType = "OMS",
+        etterbetaling = etterbetaling,
     )
 }
 
 internal data class SamordneVedtakRequest(
     val pid: Folkeregisteridentifikator,
     val vedtakId: Long,
+    val sakId: Long,
     val virkFom: LocalDate?,
     val virkTom: LocalDate?,
     val fagomrade: String,
     val ytelseType: String,
+    val etterbetaling: Boolean,
```

Comment:
perioden for etterbetaling er ikke interessant?

Reply:
Vi skal ha samme logikk rundt etterbetaling i SAM som Pesys har, og det er ikke noe sjekk på perioden der heller - bare et enkelt flagg fra vedtaket i Pesys. Men - kan være at vi skal gjøre en vurdering på vår side @HelgeJanbu?